storybook/src/Storybook/FigmaImagesProxyModel.qml

```diff
@@ -0,0 +1,30 @@
+import QtQuick 2.14
+
+ListModel {
+    /* required */ property FigmaLinksCache figmaLinksCache
+    property alias sourceModel: d.model
+
+    readonly property Instantiator _d: Instantiator {
```

Comment:
Why is the private object exposed in the public API?

Reply:
Because QtObject has no default property like e.g. `Item` and you cannot write just 
```
Instantiator {
   id: d
   ...
}
```
Convention for that is using `_d` name. I hope that name is scary enough to not be accessed from outside ;) Of course ideally it shouldn't be accessible at all.

Note that even `QtObject { id: d }` defined in `Item` is in fact accessible via public API of `Item`. Via `data` property :)